pr-demo.js

```
@@ -3,75 +3,114 @@
+// 7. Duplicate function names
 function fetchData(cb){
-  fs.readFile('data.txt', 'utf-8', (err, data) => {
-    if(err){
-      console.error("Error!", err)
-      return
-    }
-    cb(data)
-  })
+  fs.readFile('data.txt','utf8',(e,d)=>{if(e) return; cb(d)})
+}
+function fetchData(url, cb){  // overload that never gets called
```

Biome (1.9.4)

[error] Shouldn't redeclare 'fetchData'. Consider to delete it or rename it.

'fetchData' is defined here:


(lint/suspicious/noRedeclare)